ajax/libs/ami.js/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "ami.js",
+  "npmName": "ami.js",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "ami.min.js",
+  "version": "v0.0.10",
```

Comment:
was follow https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/three.js/package.json which has "r" in the version